internal_transfers/actions/src/utils.ts

```diff
@@ -1,4 +1,29 @@
-import { TransferEvent } from "./models";
+import { TokenImbalance, TransferEvent } from "./models";
+
+export function aggregateTransfers(
+  transfers: TransferEvent[],
+  focalAccount: string
+): TokenImbalance[] {
+  let accumulator = new Map<string, bigint>();
+  transfers.map((transfer) => {
+    const { to, from, amount, token } = transfer;
+
+    const currVal = accumulator.get(token) ?? 0n;
```

Comment:
Out of curiosity: what is the `?? 0n` at the end for?

Reply:
I assume because the accumalor can return None?

Reply:
Yea and here we are setting the default value to 0. Its weird because `0n` is the way to define the JS primitive `bigint`  type.